doc/Introduction.md

```diff
@@ -1 +1,81 @@
 
+----------
+## RocketMQ Client Go
+
+### 1. Go Version
+* go1.11.1 darwin/amd64
+
+### 2. Dependency
+* [librocketmq](https://github.com/apache/rocketmq-client-cpp)	
+
+### 3. Build and Install
+#### macOS Platform (macOS Mojave 10.14)
+* Install Compile tools (homebrew package manager)
+    ```
+    1. xcode-select --install
+    3. brew install cmake
+    4. brew install automake
+    ```
+* Install dependencies
+    1. [Go official download](https://golang.org/dl/)
+
+    2. Get go client package
+    ```
+    go get github.com/gaufung/rocketmq-client-go
```

Comment:
Please change to github.com/apache/rocketmq-client-go

Reply:
I have fixed it